README.md

```diff
@@ -1,97 +1,161 @@
+  graph.insertEdge(parent, null, 'a regular edge', vertex01, vertex02);
+});
 ```
 
-and select `@mxgraph/html`.
+You will see something like in the following:
 
-Since both commands are in watch mode, so it's recommended to open two terminals and run them separately. When a file is saved from the core package, the html storybook will be automatically updated.
+![maxGraph panning demo](docs/maxgraph_demo.gif "maxGraph panning demo")
 
-===
+For more details, have a look at the [storybook stories](packages/html/stories).
 
-# Looking for maintainers
 
-12 Nov 2020.
+## Support
 
-If you are interested in becoming a maintainer of mxGraph please comment on issue #1 https://github.com/jsGraph/mxgraph/issues/1
+For usage question, please open a new [discussion](https://github.com/maxGraph/maxGraph/discussions/categories/q-a) on GitHub. You can also use
+[GitHub discussions](https://github.com/maxGraph/maxGraph/discussions) for other topics like `maxGraph` development or to get the latest news.
 
-Initial objectives:
+Until we provide a complete documentation, you can check the mxGraph resources
+- mxGraph documentation: [Github pages branch](https://jgraph.github.io/mxgraph/). The key resources are the JavaScript user manual, the JavaScript examples and the JavaScript API specification.
+- [mxgraph tag on Stack Overflow](http://stackoverflow.com/questions/tagged/mxgraph). Please ensure your questions adhere to the [SO guidelines](http://stackoverflow.com/help/on-topic), otherwise it is likely to be closed.
 
-- The first priority is to maintain a working version of mxGraph and its **npm package**
-- The ambitious stretch goal is to refactor the codebase to create a modern modular, tree shakable, version of mxGraph to reduce the whole package size.
 
--- Colin Claverie
+## Migrating from mxGraph
 
-Note that the original default branch was `master`, and this has now been renamed `main`. If you had a checkout with the old branch name then follow these instructions to get the new branch name:
+**NOTE**: the following is only an indication of the migration. A more complete guide will be written in the future.
 
-```
-git branch -m master main
-git fetch origin
-git branch -u origin/main main
-git remote set-head origin -a
-```
+`maxGraph` APIs are not fully compatible with mxGraph APIs. The concepts are the same, so experienced mxGraph users should be able to switch from mxGraph to maxGraph without issues.
 
-# Transition to Apache 2.0 license
+The main changes is the Internet Explorer support removal (including VML support).
 
-This project is published under the terms of a [Modified-Apache-2.0](LICENSE) license and currently preparing to move back to the standard Apache 2.0 license.  
-From now on, all contributions to this project must be under the terms of the Apaches 2.0 license as stated in https://www.apache.org/licenses/LICENSE-2.0.  
-See https://github.com/maxGraph/maxGraph/issues/89 for more details.
+### Migrating JavaScript
```

Comment:
❓ please provide suggestion for the JS and TS migration title. I'm out of ideas.